universe/interfaces.go

```diff
@@ -164,6 +164,8 @@ type Node interface {
 	AddAPIRegister(register APIRegister)
 
 	WriteInfluxPoint(point *influxWrite.Point) error
+
+	GetGuestUserTypeID() (uuid.UUID, error)
```

Comment:
yes, it's much better to move it to our helpers to avoid polluting of our interfaces

Reply:
Moved to helper
`helper.GetGuestUserTypeID()`